### DISCUSSION TITLE
Admin Dashboard via Cloudflare

### DISCUSSION THREAD

**Discussion by User00092** [pocketbase/pocketbase#3658] [Q&A]
I'm having an issue with the admin dashboard returning a blank screen, as shown in the image below. Accessing the raw IP of the server, it works fine. I believe this is a me issue, however, unable to find a solution, I'm seeking advice/help here.  

Information:
1. I'm using apache2 to act as a reverse proxy.
2. The page return status code 200

Here is the apache2 config:
```
<VirtualHost *:443>
        ServerName db.[URL]:443

        # Reverse Proxy
        ProxyPass / http://localhost:8090/ nocanon
        ProxyPassReverse / http://localhost:8090/

        # SSL & HTTPS

        SSLEngine on
        SSLCertificateFile [PATH]
        SSLCertificateKeyFile [PATH]
</VirtualHost>
```

Through cloudflare:
![Screenshot 2023-11-01 184959](https://github.com/pocketbase/pocketbase/assets/103274860/ab6b6487-3a21-4e49-9f02-64db84d9ebea)

Via IP:
![Screenshot 2023-11-01 191401](https://github.com/pocketbase/pocketbase/assets/103274860/6c7e9906-a390-438b-b7ae-50bb0cab154b)

Cloudflare "config":
![Screenshot 2023-11-01 191528](https://github.com/pocketbase/pocketbase/assets/103274860/08299053-2720-440a-a1c7-d29afe24189f)

![Screenshot 2023-11-01 191218](https://github.com/pocketbase/pocketbase/assets/103274860/34f5b966-09f5-4dec-ae13-a63b6d41b217)

NOTE: I would like to note that the api seems to work fine through cloudflare.

**Comment by ganigeorgiev** [comment]
Could you open the browser's Inspect tool and check what error is being returned?
The Admin UI is a static client-side application so I suspect that probably the html/js files are not being served correctly for some reason.

**Reply by User00092** [reply]
I apologize for the delayed response. I've attached the image below. It seems to be blocked due to "disallowed MIME type".

![Screenshot 2023-11-02 165442](https://github.com/pocketbase/pocketbase/assets/103274860/883eab89-c99f-47b8-9891-4eae0d80e969)

**Comment by User00092** [comment]
I found the issue. I have the path "*/assets*" blocked, which in return blocks "internal" traffic as well, therefore blocking access to the resource.

I resolved this issue by simply allowing access to the path. Thank you for your time.
